recipe-client-addon/lib/ShieldRecipeClient.jsm

```diff
@@ -62,7 +63,7 @@ this.ShieldRecipeClient = {
       () => Preferences.ignore(PREF_LOGGING_LEVEL, LogManager.configure),
     );
     log = LogManager.getLogger("bootstrap");
-
+/* FIXME this is preventing first run.
```

Comment:
@mythmon mentioned that there is a PR to remove this bit, if we land and I rebase this can go away.